eclipse.html.slim

```diff
@@ -0,0 +1,166 @@
+      td 
+       p Fred Bricon
+       a href = "https://twitter.com/fbricon" 
+        i.fa.fa-lg.fa-twitter-square
+      td
+       p m2e-wtp provides a set of m2e connectors used for the configuration of Java EE projects in WTP, brings unique Maven features to your Eclipse IDE and helps you convert your Eclipse projects to Maven.
+     tr
+      td
+       a href = "https://www.eclipse.org/swtbot/" 
+        img(src="https://www.eclipse.org/swtbot/images/backgroundMain.png" alt="SWTBot-Project-Logo" width="200" height="250")
+      td 
+       p Mikael Istria
+       a href = "https://twitter.com/mickaelistria" 
+        i.fa.fa-lg.fa-twitter-square
+      td
+       p SWTBot is an open-source Java based UI/functional testing tool for testing SWT, Eclipse and GEF based applications.
+     tr
+      td
+       a href = "https://www.eclipse.org/thym/" THyM
+      td 
+       p Görkem Ercan
+       a href = "https://twitter.com/gorkemercan" 
+        i.fa.fa-lg.fa-twitter-square
+      td
+       p The Hybrid Mobile project enables cross platform mobile development with Eclipse IDE. It delivers IDE components and a framework for developing Hybrid Mobile Applications on the Eclipse platform.
+     tr
+      td
+       a href = "https://projects.eclipse.org/projects/rt.vertx" 
+        img(src="http://vertx.io/logo-white-big.png" alt="Vert.x-Project" width="200" height="250")
+      td 
+       p Tim Fox
+       a href = "https://twitter.com/timfox" 
+        i.fa.fa-lg.fa-twitter-square
+      td
+       p Vert.x is a toolkit for building reactive applications on the JVM
+
+.row	
+ p
+  .large-5.column.center
+   p &nbsp;
+  .large-19.column
+    h3 But There Is Even More To This Partnership
+    p Beside the individual project contributions, Red Hat is also engaged with the Eclipse Foundation and helps on many different levels with governing the platform.
+    table
+     tr
+      td width = 200
+       p Member in the Eclipse Board Of Directors
+      td width = 200
+       p Max Rydahl-Andersen Red Hat, Inc.
+      td
+       p Max Rydahl Andersen is leading the development of JBoss Tools and Red Hat JBoss Developer Studio and have several years working with Eclipse from both outside and inside of eclipse.org. His main interests for Eclipse.org is to keep the Eclipse IDE a viable platform for building desktop based tooling while also getting involved in offering a complementary cloud and web-based tooling offering.
+     tr 
+      td 
+       p Member in the Eclipse Project Management Committee
+      td
+       p Aleksandar Kurtakov, Red Hat Inc.
+      td
+       p Aleksandar is part of Red Hat's Eclipse team. He is Platform SWT committer, Tools PMC member and leads the Linux Tools project. He is very interested in deeper native platform integration both as runtime experience and as distribution channel. He is also interested in growing the contributors and better aligning with modern systems.
```

Comment:
I'm not sure what you mean by 'deeper native platform integration both as runtime experience and as distribution channel.' It doesn't seem to read right, but I don't understand the statement, so can't suggest an improvement.